packages/SettingsLib/src/com/android/settingslib/bluetooth/BluetoothEventManager.java

```diff
@@ -270,6 +285,10 @@ public void onReceive(Context context, Intent intent, BluetoothDevice device) {
             }
             // Inform CachedDeviceManager that the adapter state has changed
             mDeviceManager.onBluetoothStateChanged(state);
+            if (state == BluetoothAdapter.STATE_ON)
+                BluetoothTimeoutReceiver.setTimeoutAlarm(context,
+                        Settings.Global.getLong(context.getContentResolver(),
+                                Settings.Global.BLUETOOTH_OFF_TIMEOUT, 0));
```

Comment:
Please use braces for this if statement since we're following AOSP's code style: https://source.android.com/setup/contribute/code-style#use-standard-brace-style

Reply:
Thx, done